newsheadlines.py

```diff
@@ -0,0 +1,63 @@
+import click
+import requests
+#not in range expection error
+class NotInRangeError(Exception):
+    pass
+
+def news_source_menu():
+    click.echo(click.style('Select your favorate source of News', fg='blue'))
```

Comment:
Build the string using python string formatting and call `click.echo` once to print out the string